sklearn/tree/_classes.py

```diff
@@ -1021,6 +1021,7 @@ def predict_log_proba(self, X):
     )
     @property
     def n_features_(self):
+        """int: The number of features seen during `fit`."""
```

Comment:
@thomasjpfan `napoleon` mentions that you can do the following for a property. Numpydoc raises an error because the summary does not start with a capital letter (since this is the type). However, it also mentioned that this is probably a property. So I assume this is kind of valid.

Do you have any thoughts on that?

Reply:
The following will work:

```
"""int : The number of features seen during `fit`.
"""
```

But `black` will reformat it on a single line ;P

Reply:
what is `napoleon`?

Reply:
I agree with black: for oneliner docstrings, PEP257 says we should put the quotes on the same line.

Reply:
It should be possible to ignore GL02 by adding the following in `conf.py`:

```python
numpydoc_validation_checks = {"all", "GL02"}
```

https://numpydoc.readthedocs.io/en/latest/validation.html#docstring-validation-during-sphinx-build

Reply:
Basically, we want to ignore GL02 only in properties.